### ISSUE TITLE
Docs menu not clickable with keyboard navigation

### ISSUE THREAD

**Issue by janosdebugs** [opentofu/opentofu.org#259]
When using the website with a keyboard (e.g. when using the NVDA screen reader) it is not possible to activate the Docs menu with the keyboard by pressing space or enter. On behalf of the people not able to use a mouse, this should be fixed.

On a related note, if you are a screen reader user, I'd like to hear from you how you consume code-heavy content on the Internet, please comment/reach out!

**Comment by namgyalangmo** [comment]
Could you please provide more details? Are there any relevant files or code that I can look into to investigate this issue further? Any additional information would be greatly appreciated!

**Comment by janosdebugs** [comment]
Hey @namgyalangmo basically, visit the website and navigate to the CLI docs without using the mouse. This currently doesn't work. Unfortunately, I have no more insights into why the menu doesn't work.

**Comment by namgyalangmo** [comment]
@janosdebugs I tried navigating without mouse via Voiceover on a Mac device, it seems to work as expected over there, is the issue specific to NVDA or a windows device?

**Comment by janosdebugs** [comment]
Hey @namgyalangmo thanks for testing it, seems like the recent Docusaurus 3 update fixed the issue!
